.github/workflows/release-sdk.yml

```diff
@@ -3,15 +3,66 @@ name: Build and Release W&B SDK
 on:
   workflow_dispatch:
     inputs:
-      use_pre:
+      version:
+        type: string
+        description: "version to assign to the release"
+        required: true
+      update-changelog:
         type: boolean
-        description: "pre-release"
+        description: "update the changelog and release notes"
         required: false
         default: false
 
 jobs:
+  prepare-release:
+    name: Prepare Release
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+
+      - name: Setup git config
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
looks like magic. Where do we find this email?

Reply:
yeah, we found it somewhere a while back, need to figure out where and add documentation